assets/intasend-inline.js

```diff
@@ -0,0 +1,177 @@
+    return check;
+};
+
+module.exports = {
+    MobileCheck
+};
+},{}],2:[function(require,module,exports){
+let agents = require("./agents")
+
+const IntaSend = {
+    _redirectURL: "",
+    _publicAPIKey: "",
+    _element: "tp_button",
+    _btnElement: Object,
+    _dataset: Object,
+    _live: true,
+    setup(obj) {
+        IntaSend._publicAPIKey = obj.publicAPIKey
+        IntaSend._redirectURL = obj.redirectURL
+        IntaSend._live = obj.live
+        IntaSend._element = obj.element || IntaSend._element
+        IntaSend.is_mobile = agents.MobileCheck()
+
+        IntaSend._btnElement = document.getElementsByClassName(IntaSend._element)
+        if (IntaSend._btnElement) {
+            for (let i = 0; i < IntaSend._btnElement.length; ++i) {
+                let btn = IntaSend._btnElement[i]
+                btn.addEventListener('click', function () {
+                    let dataset = btn.dataset
+                    IntaSend.loadPaymentModal(dataset)
+                })
+            }
+        }
+
+        // HANDLE NEW MESSAGE NOTIFICATIONS
+        function bindEvent(element, eventName, eventHandler) {
+            if (element.addEventListener) {
+                element.addEventListener(eventName, eventHandler, false);
+            } else if (element.attachEvent) {
+                element.attachEvent('on' + eventName, eventHandler);
+            }
+        }
+
+        bindEvent(window, 'message', function (e) {
+            if (e.data.message) {
+                if (e.data.message.identitier == 'intasend-status-update-cdrtl') {
+                    if (e.data.message.state === "COMPLETE") {
+                        if (IntaSend._redirectURL) {
+                            window.location.href = IntaSend._redirectURL
+                        }
+                    }
+                } else if (e.data.message.identitier == 'intasend-close-modal-cdrtl') {
+                    IntaSend.clearElements()
+                }
+            }
+        });
+
+
+        // End message events
+
+        return IntaSend
+    },
+    run(obj) {
+        IntaSend.loadPaymentModal(obj)
+    },
+    loadPaymentModal(dataset) {
+        dataset.callback_url = IntaSend._redirectURL
+        dataset.public_key = IntaSend._publicAPIKey
+        dataset.host = window.location.protocol + "//" + window.location.host
+        dataset.is_mobile = IntaSend.is_mobile
+
+        IntaSend.clearElements()
+        let modalContent = IntaSend.prepareModal()
+        if (!IntaSend.is_mobile) {
+            IntaSend.closeModalIcon(modalContent)
+        }
+        let iframe = IntaSend.prepareFrame(modalContent, dataset)
+        return iframe
+    },
+    clearElements() {
+        let iframes = document.querySelectorAll('iframe');
+        for (let i = 0; i < iframes.length; i++) {
+            iframes[i].parentNode.removeChild(iframes[i]);
+        }
+        // Remove modals
+        let modals = document.querySelectorAll('modal');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/IntaSend/woocommerce-plugin/pullRequest?prid=7551924)
```suggestion
        let modals = document.querySelectorAll("modal");
```